src/allmydata/test/test_runner.py

```diff
@@ -167,7 +181,7 @@ def do_create(self, kind, *args):
 
         n1 = os.path.join(basedir, command + "-n1")
         argv = ["--quiet", command, "--basedir", n1] + list(args)
-        rc, out, err = yield run_cli(*argv)
+        rc, out, err = yield run_cli(*map(unicode_to_argv, argv))
```

Comment:
So I thought about this some more, and I think this is a mistake. In particular, CLIs are never run by actual code, only tests, and so it's better to just be lenient in one place, the API that runs a CLI, rathe than trying to fix every call in every test.

Reply:
In this particular case it doesn't seem worth fixing though, given it's just a couple instances, but to enable easy porting of `test_create.py` I fixed the underlying function to do the unicode_to_argv.

Reply:
I had the same thought, and since this was my first 2to3 conversion, I wanted to be conservative with the changes, but I fully expected to eventually clean up the unicode sandwich. Ultimately, I'd like to see less repetition in CLI code and just have a clean interface that maybe only supports unicode. Regardless, I'm happy with your proposed approach.